below/view/src/cgroup_view.rs

```diff
@@ -266,6 +269,12 @@ impl CgroupView {
                 tab: &*CGROUP_CPU_TAB,
             },
         );
+        tabs_map.insert(
+            "Pids".into(),
+            CgroupView {
+                tab: &*CGROUP_PIDS_TAB,
+            },
+        );
```

Comment:
Again, I'd prefer PidsMax here as part of the Properties model in the Properties tab

Reply:
Yep, moved Max to props, but kept Current in this tab.